### PR TITLE
ugly "fix" to prevent "buffer overflow detected" errors when the LCD …

### PR DESCRIPTION
**Pull request by ghost**
…width is bigger than 16 and device-name as well

### CODE REVIEW

File: clients/lcdproc/disk.c [hunk 1]
```diff
@@ -59,7 +59,7 @@ disk_screen(int rep, int display, int *flags_ptr)
 
 	/* Holds info to display (avoid recalculating it) */
 	struct disp {
```
Comment (haraldg) [inline clients/lcdproc/disk.c:61]: This function seems broken in many ways. Most notably table is not static, so it is recalculated anyway.

Maybe just merge the two for loops together and get rid of table completely?

**Comment by haraldg** [comment]
@vintagepc Maybe this PR addresses the crash you are seeing?

**Comment by vintagepc** [comment]
Ooh, that could definitely be it, the disk definitely has a path longer than 16 chars and the display is a 40x4

**Comment by haraldg** [comment]
Fixed by #181

**Pull request closed without merging**